src/Components/BlogPost/BlogPostNavigation.js

```diff
@@ -33,12 +33,22 @@ const BlogPostNavigation = Scrivito.connect(({ currentPost }) => {
 const BlogPostNextLink = Scrivito.connect(({ currentBlogPost }) => {
   const currentDate = currentBlogPost.get("publishedAt");
 
-  // find greater than publishedAt
-  const newerPost = Scrivito.getClass("BlogPost")
-    .where("publishedAt", "isGreaterThan", currentDate)
-    .order("publishedAt", "asc")
+  const sameDateNextBlogPost = Scrivito.getClass("BlogPost")
+    .where("publishedAt", "equals", currentDate)
+    .and("_id", "isGreaterThan", currentBlogPost.id())
+    .order(["_id", "asc"])
```

Comment:
* [ ] This would sort by by `_id` and `asc`
```suggestion
    .order("_id", "asc")
```